style.css

```diff
@@ -0,0 +1,135 @@
+:root{
+    --border-color:#d1d1d1;
+    --table-element-color:#fb923c;
+    --primary-color: #94a3b8;
+    }
+
+/*cart-page*/
+
+
+.navbar{
+    position: sticky;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 9;
+}
+
+.nav{
+    padding: 2px 2vw;
+    display: flex;
+    justify-content: space-between;
+}
+
+.nav-items{
+    display: flex;
+    align-items: center;
+}
+
+.brand-logo-cart{
+    margin-left: 30rem;
+    height:23rem;
+    width:23rem;
+}
```

Comment:
You should avoid using direct uncommon spacings. If it's an extraordinary situation then it's ok.

Reply:
I am facing issue with spacing and alligning the items,i need help to adjust.Lack of time to do now